### DISCUSSION TITLE
Clear cache with excluded tags

### DISCUSSION THREAD

**Discussion by tigusigalpa** [laravel/framework#43150] [Ideas]
Hi all!
I think the great feature is about make artisan `cache:clear` command with excluded tags to clear, for example:
`php artisan cache:clear --exclude-tags=production,api`

**Comment by henzeb** [comment]
Myself I am not a big fan of tags. I always create seperate stores. That way I can clear caches like `php artisan cache:clear horizon` or `php artisan clear:cache api`.

**Reply by tigusigalpa** [reply]
yes, this is a good solution, but i think Laravel has to have this feature for some developers who use tags)

**Reply by henzeb** [reply]
Is this the opposite of the current --tags feature?

**Reply by tigusigalpa** [reply]
yes)
